src/Facebook/InstantArticles/Transformer/Rules/InteractiveInsideParagraphRule.php

```diff
@@ -1,4 +1,4 @@
-<?hh //decl
+<?hh // strict
```

Comment:
Hey, this rule was deprecated, since Interactive accepts Paragraph as a context. Why is it back?

Reply:
Here's the original PR for reference #186

Reply:
Class was still there, but not being used, indeed.

Reply:
Issue created: #284